report-hazard/index.html

```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <script src="script.js" defer></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.js"></script>
+    <link
+      href="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.css"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      href="https://unpkg.com/leaflet@1.9.4/dist/leaflet.css"
+      integrity="sha256-p4NxAoJBhIIN+hmNHrzRCf9tD/miZyoHS5obTRR9BMY="
+      crossorigin=""
+    />
+    <script
+      src="https://unpkg.com/leaflet@1.9.4/dist/leaflet.js"
+      integrity="sha256-20nQCchB9co0qIjJZRGuk2/Z9VM+kNiyxNV1lvTlZBo="
+      crossorigin=""
+    ></script>
+    <title>Document</title>
+  </head>
+
+  <body>
+    <nav>
+      <ul>
+        <li><a title="Select the location" href="#step1">1</a></li>
+
+        <li>
+          <a title="What type of hazard would you like to report?" href="#step2"
+            >2</a
+          >
+        </li>
+
+        <li>
+          <a title="What type of wild animal did you spot?" href="#step3">3</a>
+        </li>
+
+        <li>
+          <a
+            title="Would you like to mention any further details?"
+            href="#step4"
+            >4</a
+          >
+        </li>
+
+        <li>
+          <a
+            title="Please upload photos to help others see the reported hazard visually"
+            href="#step5"
+            >5</a
+          >
+        </li>
+
+        <li>
+          <a title="Review the report before submitting." href="#step6">6</a>
+        </li>
+
+        <li><a href="#step7">7</a></li>
+      </ul>
+    </nav>
+
+    <main>
+      <form id="form">
+        <section class="page" id="step1">
+          <h1>1. Select the location.</h1>
+
+          <div id="map"></div>
+
+          <label>Your current location is:</label>
+          <input type="text" id="name" required />
+
+          <a href="#step2">Continue</a>
+        </section>
+        <section class="page" id="step2">
+          <h1>2. What type of hazard would you like to report?</h1>
+          <input type="radio" name="category" value="Wildlife" />Wildlife
```

Comment:
The radio buttons need to have a label tag with the for attribute, so they are accesible to screen readers and click on the label tag

 <div>
    <input type="radio" id="huey" name="drone" value="huey" checked />
    <label for="huey">Huey</label>
  </div>
  
  https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/radio

Reply:
commit 97164971a362654dae737ea8bb63c8546191fc93